joy2twist/include/joy2twist/joy_input.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
```

Comment:
```suggestion
#ifndef JOY2TWIST_JOY_INPUT_HPP_
#define JOY2TWIST_JOY_INPUT_HPP_
```
We are using standard header guards